app/Http/Controllers/AirlineDestroyController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Airline;
+use Illuminate\Http\JsonResponse;
+
+class AirlineDestroyController
```

Comment:
DeleteAirlineController is better
Destroy vs delete is just preference, but the verb should go first, as otherwise they will all start with the same model name, and it is harder to differentiate them

Reply:
applies to every file :/